articles/users/concepts/overview-user-account-linking.md

```diff
@@ -168,25 +168,15 @@ The [Auth0 Node.js SDK for APIv2](https://github.com/auth0/node-auth0/tree/v2) i
 
 ## Scenarios
 
-Here are some scenarios that implement account linking:
-* [Automatic account linking](#automatic-account-linking): automatically link accounts with the same email address
+Here are two scenarios that implement account linking:
+
 * [User-initiated account linking](#user-initiated-account-linking): allow your users to link their accounts using an admin screen in your app
 * [Suggested account linking](#suggested-account-linking): identify accounts with the same email address and prompt the user in your app to link them
 
 ::: warning
```

Comment:
I think this warning is not required, as with user-initiated account linking, you don't need verified emails